server/gokbg3/grails-app/controllers/org/gokb/GlobalSearchController.groovy

```diff
@@ -43,13 +45,12 @@ class GlobalSearchController {
 
         log.debug("Using index ${grailsApplication.config.gokb?.es?.index /*?: 'gokbg3dev (auto)'*/}")
 
-        SearchRequestBuilder es_request = esclient.prepareSearch("globalSearch")
-            .setIndices(grailsApplication.config.gokb?.es?.index ?: "gokbg3")
-            .setTypes(grailsApplication.config.globalSearch.types ?: "component")
-            .setSize(result.max)
-            .setFrom(result.offset)
-            .setQuery(esQuery)
-            .addAggregation(
+        SearchRequest es_request = new SearchRequest(grailsApplication.config.gokb?.es?.index ?: "gokbg3")
```

Comment:
I'm not really a fan of hard-coded fallback names. An installation might use another index name, where this leads to exceptions. I suggest to create another ticket for this. Shall I?